src/graphql/resolvers/OrganizationResolvers.ts

```diff
@@ -4,10 +4,24 @@ async function getOrganizationsWithOptions(root, args) {
   return OrganizationDao.getWithOptions(args.request);
 }
 
+async function mutateOrganization(obj, args, context) {
+  const { organizationId } = context;
+  const { request } = args;
+  console.log(organizationId);
+  console.log(request.id);
+
+  if (organizationId !== Number(request.id)) throw new Error("Organization does not match user profile");
+
```

Comment:
Why do you even bother passing the organization id in the request ? Its like authorId

Reply:
Checking if it's the right organizationid

Reply:
Checking if it's the right organizationid